webview-ui/src/styles/accessibility.css

```diff
@@ -0,0 +1,30 @@
+/**
+ * Screen reader only utility class
+ * Visually hides content while keeping it accessible to screen readers
+ */
+.sr-only {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
+	white-space: nowrap;
+	border-width: 0;
+}
+
+/**
+ * Variant that becomes visible when focused
+ * Useful for skip links and other keyboard navigation aids
+ */
+.sr-only-focusable:focus {
+	position: static;
+	width: auto;
+	height: auto;
+	padding: inherit;
+	margin: inherit;
+	overflow: visible;
+	clip: auto;
+	white-space: normal;
+}
```

Comment:
I think we could just use the built-in [sr-only](https://v1.tailwindcss.com/docs/screen-readers) classname from tailwind, and configure it in webview-ui/src/theme.css if we want so it'd work across the webview 😄